ocis/docker/eos-ocis/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:7 as build
+ARG BRANCH=master
+
+ENV CGO_ENABLED=1
+ENV GOOS=linux
+RUN rpm --rebuilddb && yum -y install \
+    wget \
+    time \
+    make \
+    gcc \
+    git
+RUN wget -q https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz
+RUN mkdir -p /usr/local/bin
+RUN tar xf go1.14.2.linux-amd64.tar.gz -C /usr/local
+RUN ln -s /usr/local/go/bin/* /usr/local/bin
+RUN git clone https://github.com/owncloud/ocis.git
+RUN cd ocis && git checkout ${BRANCH}
+RUN cd ocis && git log -n 1
+RUN cd ocis && go build -v -o bin/ocis -toolexec '/usr/bin/time -v' ./cmd/ocis
+# RUN cd ocis && make generate bin/ocis
+
+# build user lookup test program
+RUN mkdir -p /build
+COPY user.go /build/user.go
+RUN cd /build && go build -o user user.go
```

Comment:
Codacy found an issue: [Use WORKDIR to switch to a directory](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6300846)